api/desecapi/serializers/tokens.py

```diff
@@ -73,4 +73,4 @@ def save(self, **kwargs):
         try:
             return super().save(**kwargs)
         except django.core.exceptions.ValidationError as exc:
-            raise serializers.ValidationError(exc.message_dict, code="precedence")
+            raise serializers.ValidationError(exc.message, code="precedence")
```

Comment:
undo to keep compatibility and prepare for usage in webapp

Reply:
done